scripts/source-maintenance.sh

```diff
@@ -354,6 +453,11 @@ for FILENAME in `git ls-files`; do
     fi
 
 done
+
+    run_ processDebugSections || return
+    run_ processDebugMessages || return
+
+    rm boilerplate_fix.sed
```

Comment:
This change in merged/moved code results in srcFormat killing the script when executed with `--check-and-update-copyright no`. If this code remains in its new place, then let's restore `-f`.

```suggestion
    rm -f boilerplate_fix.sed
```

However, I do not think this line should have been moved at all! It is unrelated to debug-sections and debug-messages [cleanup] changes, and the boilerplate_fix.sed file is created outside srcFormat, so it would be more logical to destroy it outside srcFormat as well. Please consider undoing this move.

Reply:
>  undoing this move.

Done.